### ISSUE TITLE
Monitoring Plan Evaluation Report

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#2275]
### Definition of Ready
- [x] Does this meet the INVEST criteria?
- [x] Does this have a clearly defined user?
- [x] Does this have an estimate?
- [x] Does this have acceptance criteria?
- [x] Does this have sufficient context for the specific functionality being developed (design, business rules, etc.)
- [x] Does this have a dev link been provided in the design?
- [x] Is this traceable to a feature, epic, or PI?

## User Story
As an ECMPS user, I want the 'Monitoring Plan Evaluation Report' to be available when a MP evaluation is complete so I can look at details about the validated data

## Acceptance Criteria
Given a Monitoring Plan Evaluation Report is available
When I access it (Click on the hyperlink for Passed/Informed Message/Critical Errors)
Then I am able to view the Report in a Modal, and 'Print Report'

Note: 'Print Report' button is available and when accessed triggers the browser's print function

Question: Shading indicating the scroll

Dependency: API endpoints to be available once the Check Engine is ready

### Does this Story require 508 Testing?
- [ ] Implementing a new authoring tool that will produce code / content for the application
- [ ] Implementing a 3rd party widget that will be part of the application
- [ ] Adding new / updating custom functionality 
- [ ] Adding new / updating a page template
- [ ] Adding new / replacing existing functionality (unless the replacement has already passed 508 testing)
- [ ] Full Site Review prior to a scheduled EPA 508 review
- [ ] Planning a video that will appear on the application
- [ ] Producing a video that will appear on the application
- [ ] Producing Training materials
- [ ] Producing documents

## Context
https://xd.adobe.com/view/05b0e234-0db4-4767-9d26-bfbe67442482-50e3/

## Testing
Test Case: [TC503](https://epacamd.testquality.com/project/11428/plan/22924/test/316211)
Tester: @mosesdeeCVP 

### Definition of Done
- [x] Have all Acceptance Criteria been agreed to and validated by the scrum team?
- [x] Has the code been unit tested and peer reviewed?
- [x] Have the functional tests been executed?
- [x] Have all defects been reviewed and dispositioned, resolved, or deferred?
- [x] Is the user story ready to be deployed to the test/staging environments?

**Comment by mosesdeeCVP** [comment]
![2275 Monitoring Plan Evaluation Report - 1.png](https://images.zenhubusercontent.com/5f807025564a417cbcdf6d69/638fc5a9-0270-441d-a7ca-eb21ac55c9d9)

**Comment by mosesdeeCVP** [comment]
![2275 Monitoring Plan Evaluation Report - 2.png](https://images.zenhubusercontent.com/5f807025564a417cbcdf6d69/583e1a1e-fa5d-43f3-b779-82bac9ef139e)

**Comment by mosesdeeCVP** [comment]
Test Passed
